community-handbook/chaosscast.md

```diff
@@ -170,22 +177,21 @@ Georg Link has an IFTT recipe to:
 
 ## Procedures
 
-These procedures are for the organizer of each episode:
+These procedures are for the organizer and coordinator of each episode:
 
 ### Scheduling an episode 
 * always
   * Think of who would make a good guest for the podcast and ask them tentatively if you may invite them
-* **Step 1:** Determine a guest and panlists
-  * **Action: **Recruit interview guest and decide topic together
-    * Ask for 60-90 minute timeslots that work for guest.
-    * Create a Doodle poll for panelists to schedule time
-  * **Action:** Prepare the planning document (copy the template and start filling in information)
-  * **Action:** send out an [invitation email](https://docs.google.com/document/d/1IniJc9o1OCBBTYv50W3WTfZYmshi4aYusHIK4ADulk8/edit) with a [Doodle](doodle.com) for finding a time
-    * Invite guest
-    * Invite panelists ([list of email addresses](https://docs.google.com/document/d/1ca9TJY17cj-cBWmJP5qjCX56133unsaOlEUwACc4PMA/edit))
+* **Step 1:** Determine a guest and panelists (Organizer)
+  * **Action:** Recruit interview guest and decide topic and content of the podcast together
+* **Step 2:** Coordinate Episode (Community Manager)
+    * **Action:** Work with guest to find a suitable time 60-90 minutes.
+    * **Action:** Find time that works for guest and organizer.
+    * **Action:** Prepare the planning document (copy the [Episode Planning Document template](https://docs.google.com/document/d/17qjAafr0aAWXV_HR-WiefHOa881t8X7tEoiZnPBh3ao/edit#heading=h.cmpmu92ylw4d) and start filling in information.)
+   * **Action:** send out an [invitation email](https://docs.google.com/document/d/1IniJc9o1OCBBTYv50W3WTfZYmshi4aYusHIK4ADulk8/edit) with a [Doodle](doodle.com) for finding a time
+   * **Action:** Invite panelists ([list of email addresses](https://docs.google.com/document/d/1ca9TJY17cj-cBWmJP5qjCX56133unsaOlEUwACc4PMA/edit))
       * Prioritize those who said they were interested in the topic
       * If needed, [invite new panelists](https://docs.google.com/document/d/1gGf9n6D5g2Y_YfqVuWOvWvZvNvKmbQr9adruHeQTAu0/edit?usp=sharing)
-* **Step 2** (about 1 week after sending invites): schedule recording
   * **Action:** Lock in date in Doodle poll
   * **Action:** Send calendar invitation for the best date of podcast
```

Comment:
```suggestion
  * **Action:** Send calendar invitation for the best date of podcast (only the guest, organizer, and panelists that will be in the episode)
```

Reply:
Ugh, I committed this without realizing I needed to sign off on it, and I'm not sure how to do this after the fact. Sorry!

Reply:
No worries, see comment below.